src/push_service/mod.rs

```diff
@@ -177,15 +177,14 @@ impl PushService {
         }
     }
 
-    #[tracing::instrument(skip(self, path), fields(path = %path.as_ref()))]
+    #[tracing::instrument(skip(self))]
     pub fn request(
         &self,
         method: Method,
         endpoint: Endpoint,
```

Comment:
Can we have some nice display implementation for `Endpoint`, for in tracing?

Reply:
Done in 4e49b1bc29b1f8f8ce08bda6aecbcc384ec967a9